test/new-language.md

```diff
@@ -0,0 +1,190 @@
+# Adding a New Language Integration Test
```

Comment:
hindsight 20/20, I should have named this `new-generator.md` and rephrased this to `# Adding a New Generator Integration Test`. 

Language is a poor choice, as really generators could be for more than just languages, could be frameworks.

Reply:
I think we could rename the file.